src/index.js

```diff
@@ -82,7 +83,16 @@ function configureInterceptors(instance, url, { username, password } = {}) {
 
   instance.interceptors.response.use(res => {
     return res.data;
+  }, err => {
+    if (!(err instanceof Error) && typeof err === "object") {
+      return Promise.reject(errorify(err));
+    }
+    return Promise.reject(err);
```

Comment:
yep that's what I mean, convert errors into httpErrors as well, and set a default statusCode like 500